documentation/modules/rn-2.5.adoc

```diff
@@ -106,9 +111,9 @@ The error status message for a VM with no operating system on the *Plans* page o
 vSphere only: Migrations from {rhv-short} and OpenStack do not fail, but the encryption key may be missing on the target {ocp} cluster.
 
 
-.Migration fails during precopy/cutover while a snapshot operation is performed on the source VM
+.Migration fails during precopy/cutover while performing a snapshot operation on the source VM
 
-Warm migration from {rhv-short} fails if a snapshot operation is performed on the source VM. If a user performs a snapshot operation on the source VM at the time when a migration snapshot is scheduled, the migration fails instead of waiting for the user’s snapshot operation to finish. link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
+Warm migration from {rhv-short} fails if a snapshot operation is triggered and running on the source VM at the same the migration is scheduled. The migration does not wait for the snapshot operation to finish. link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
```

Comment:
Please re-read this sentence 

```
... at the same the migration is scheduled
```

Reply:
Corrected

Reply:
Not corrected

Reply:
@ahadas  Please review
Warm migration from {rhv-short} fails if a snapshot operation is triggered and running on the source VM at the same time the migration is scheduled. The migration does not wait for the snapshot operation to finish.